apps/desktop/scripts/ui-shots/kimi_one_component.py

```
@@ -0,0 +1,38 @@
+import json
+import sys
+import urllib.request
+from pathlib import Path
+
+KEY = os.environ.get("MOONSHOT_API_KEY", "")
+COMPONENT = sys.argv[1]  # PlanCard | Timeline | ActionCard | ConfirmCardArea
+BASE = Path(r"D:\Desktop\811\MiQi\apps\desktop\src\renderer\features\chat\components")
+CODE = (BASE / f"{COMPONENT}.tsx").read_text(encoding="utf-8")
+b64 = base64.b64encode(Path(r"D:\Desktop\811\MiQi\docs\workbuddy-5张参考-拼图.png").read_bytes()).decode()
+
+PROMPT = f"""参考图（拼图，从左到右 5 张）是用户认可的简洁优雅 UI：大圆角白卡 16-20px、深色近黑主按钮 #1f1f1f、灰色状态小字、彩色小圆点、大量留白、克制。用户要求对照抄。
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)